Makefile

```diff
@@ -0,0 +1,121 @@
+# Requirements
+# docker or podman installed and running
+# git installed
+# .git_token file exists with current Token (or injected as env var)
+# Linux OS
+
+ifndef WORKFLOW_ID
+ifneq ($(MAKECMDGOALS),help)
+$(error WORKFLOW_ID variable is not defined. Please provide the required value.)
+endif
+endif
+
+CONTAINER_ENGINE ?= $(shell which podman >/dev/null 2>&1 && echo podman || echo docker)
+ifneq (,$(wildcard $(CURDIR)/.docker))
+	DOCKER_CONF := $(CURDIR)/.docker
+else
+	DOCKER_CONF := $(HOME)/.docker
+endif
+
+WORKDIR ?= $(shell mktemp -d)
+ifeq ($(shell uname),Darwin)
+	# Use a fixed folder to simplify limactl configuration (must be mounted with Write permissions)
+	WORKDIR := /tmp/serverless-workflows
+endif
+SCRIPTS_DIR := scripts
+
+GIT_USER_NAME ?= $(shell git config --get user.name)
+GIT_USER_EMAIL ?= $(shell git config --get user.email)
+GIT_REMOTE_URL := $(shell git config --get remote.origin.url)
+GIT_REMOTE_URL := $(shell echo "$(GIT_REMOTE_URL)" | sed 's/\.git//')
+GIT_REMOTE_URL := $(shell echo "$(GIT_REMOTE_URL)" | sed 's/git@/https:\/\//')
+GIT_REMOTE_URL := $(shell echo "$(GIT_REMOTE_URL)" | sed 's/com:/com\//')
+PR_OR_COMMIT_URL ?= "$(GIT_REMOTE_URL)/commits/$(shell git rev-parse --short=8 HEAD)"
+GIT_TOKEN ?= $(shell cat .git_token)
+
+LINUX_IMAGE ?= quay.io/orchestrator/ubi9-pipeline:latest
+
+WORKFLOW_ID := ""
```

Comment:
probably my makefile ignorance, but shouldn't it be set to `$(WORKFLOW_ID)`?

Reply:
no need for `$()` in variable declaration:
https://makefiletutorial.com/#variables

anyway, you caught a good point, because `WORKFLOW_ID` must be provided by the caller, so this line has to be removed, thanks!
```
ifndef WORKFLOW_ID
$(error WORKFLOW_ID variable is not defined. Please provide the required value.)
endif
```
PS: also the reference to the `help` target has to be removed, since I deleted it at the last minute

Reply:
why delete the `help`?

Reply:
I tried to automate the generation of the help using the comments above the targets, but this depends on the layout if the comments, so I remove it for now, but I've tracked https://github.com/parodos-dev/serverless-workflows/issues/110 for future enhancements

Reply:
ok, thank you :)